requirements-cyhy_ops.txt

```diff
@@ -0,0 +1,3 @@
+# This file contains packages the CyHy operators need for their Docker image.
```

Comment:
Just to be abundantly clear, clarify that these are pip package names.  The goal is to avoid any confusion with apt package names.
```suggestion
# This file contains PyPi (pip) packages the CyHy operators need for
# their Docker image.
```